### PR TITLE
Npm

### PR DESCRIPTION
**Pull request by gazuntype**
switch to npm.

### CODE REVIEW

File: Editor/Backend.cs [hunk 1]
```diff
@@ -123,16 +124,32 @@ public class Backend {
...
         public async Task<List<ReleaseInfo>> GetMasterReleaseInfo() {
             try {
-                using UnityWebRequest req = UnityWebRequest.Get(ReleaseURL);
+                using UnityWebRequest req = UnityWebRequest.Get(RegistryURL);
                 await req.SendWebRequest();
-                return JsonConvert.DeserializeObject<List<ReleaseInfo>>(req.downloadHandler.text);
+                JObject jsonResult = JObject.Parse(req.downloadHandler.text);
+                JToken versions = jsonResult["versions"];
+                if (versions == null) {
+                    Debug.LogError("Could not find versions");
```
Comment (chornesays) [inline Editor/Backend.cs:132]: Seems like we should print something to the status bar here too

File: Editor/Backend.cs [hunk 2]
```diff
@@ -123,16 +124,32 @@ public class Backend {
...
+                    JToken release = version["release"];
+                    if (release == null) {
+                        continue;
+                    }
+                    ReleaseInfo info = release.ToObject<ReleaseInfo>();
+                    releaseInfos.Add(info);
+                }
+                return releaseInfos;
             } catch (Exception e) {
                 Debug.LogError(e.Message);
```
Comment (chornesays) [inline Editor/Backend.cs:148]: would also love a status bar print here

File: Editor/DevPanel.cs [hunk 3]
```diff
@@ -7,6 +7,7 @@
 using System.IO;
 using Filta.Datatypes;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
```
Comment (chornesays) [inline Editor/DevPanel.cs:10]: eventually would love to rip out linq (i think its used elsewhere as well)

I dont care right now. move fast

**Review by chornesays — APPROVED** [review]
Dude. Amazing. I love this so much. As soon as we ship ill be ripping out the git part of the getting started doc

File: Editor/DevPanel.cs [hunk 4]
```diff
@@ -22,6 +23,10 @@ public class DevPanel : EditorWindow {
         private const string PublishPageLink = "https://www.getfilta.com/mint";
         private const string RunLocallyMenuName = "Filta/(ADVANCED) Use local firebase host";
         private const string TestEnvirMenuName = "Filta/(ADVANCED) Use test environment (forces a logout)";
+
+        private const string RegistryName = "Filta Artist Suite";
+        private const string RegistryUrl = "https://registry.npmjs.org";
```
Comment (cdiddy77) [inline Editor/DevPanel.cs:28]: This feels very official. Npm is the big show, sort of.

**Pull request closed without merging**